.dockerignore

```diff
@@ -0,0 +1,15 @@
+.env.template
```

Comment:
Another file we'll definitely want to ignore is the `.env` file, since we don't want that baked into our images when we start hosting them

Also I don't think very important with what you've got, but worth noting that unlike `.gitignore`, `<filename>` here only ignores files named that at the root of the repo, not in subfolders, so for sensitive files I'd recommend `**/.env` over just `.env`